test/sqlancer/clickhouse/ast/ClickHouseBinaryComparisonOperationTest.java

```diff
@@ -0,0 +1,130 @@
+package sqlancer.clickhouse.ast;
+
+import org.junit.jupiter.api.Test;
+import ru.yandex.clickhouse.domain.ClickHouseDataType;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ClickHouseBinaryComparisonOperationTest {
```

Comment:
Great, thanks for including these tests!

Reply:
That is almost nothing. Used it to debug my bugs